embedded-graphics/CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ Embedded Graphics is a `no_std` library for adding graphics features to display
 
 ## [Unreleased] - ReleaseDate
 
+### Fixed
+
+- [#262](https://github.com/jamwaffles/embedded-graphics/pull/262) `Triangle`s with a fill color but no stroke color are now rendered instead of not rendered at all.
+- [#262](https://github.com/jamwaffles/embedded-graphics/pull/262) A triangle with a different stroke and fill color now renders its left-most border with the stroke color, instead of fill color
+- [#262](https://github.com/jamwaffles/embedded-graphics/pull/262) Triangles with a stroke width of 0 no longer draw the triangle outline and draw nothing as intended.
```

Comment:
This also applies to lines.

Should a change like this be considered breaking? It doesn't break the API itself, but the changed drawing behavior can cause problems. If someone used a style with only the `stroke_color` set to draw lines they would disappear after applying this PR.

Reply:
I added a mention to lines in here too. We could bikeshed whether it's breaking or not, but IMO this is fixing a bug so let's leave it non-breaking. Relying on buggy behaviour isn't the smartest thing to do anyhow ;)